spec/integ/matrix-client-event-timeline.spec.js

```diff
@@ -566,14 +607,13 @@ describe("MatrixClient event timelines", function() {
                     };
                 });
 
-            const timelinePromise = client.getEventTimeline(timelineSet, EVENTS[0].event_id);
-            await httpBackend.flushAllExpected();
-
-            const timeline = await timelinePromise;
-            expect(timeline).toBeUndefined();
+            return Promise.all([
+                expect(client.getEventTimeline(timelineSet, EVENTS[0].event_id)).resolves.toBeUndefined(),
+                httpBackend.flushAllExpected(),
+            ]);
         });
 
-        it("should return undefined when event is within a thread but timelineSet is not", async () => {
+        it("should return undefined when event is within a thread but timelineSet is not", () => {
```

Comment:
We can update this to match the pattern of the test name above (outlined in https://github.com/matrix-org/matrix-js-sdk/pull/2444#discussion_r893005753)